src/thank_you_wonder_mail.c

```diff
@@ -54,9 +56,24 @@ static EWRAM_DATA_2 WonderMailStruct_203B2C4 *sUnknown_203B2C4 = {0};
 #define PROCESS_THANK_YOU_PASSWORD 0x27
 #define THANK_YOU_PASSWORD_WRONG 0x28
 
+enum menuActions {
+    CANCEL_ACTION,
+    SEND_THANK_YOU_MAIL_ACTION,
+    GET_THANK_YOU_MAIL_ACTION,
+    // 3 - GAME_LINK
```

Comment:
If some of these missing enums are known, perhaps they can be added even if they aren't used yet.